source/_integrations/sensor.mqtt.markdown

```diff
@@ -106,7 +106,7 @@ device:
       required: false
       type: string
 device_class:
-  description: The [type/class](/integrations/sensor/#device-class) of the sensor to set the icon in the frontend.
+  description: The [type/class](/integrations/sensor/#device-class) of the sensor to set the icon in the frontend. This needs to be set to make it show on the area card, i.e 'temperature', 'humidity' etc, otherwise it fails to show.
```

Comment:
Devices classes are used for a lot more things; I think it is odd to mention this one and not the other 50+ cases. For that reason, I do think this does not belong in this part of the documentation.

Going to friendly decline this PR for that reason.

../Frenck